### PR TITLE
Multi-arch Operator Release Changes

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY

Related to https://github.com/ansible/awx-operator/pull/1681

* Adds a make target for bulding multi-arch AWX images using dockerx.

##### ISSUE TYPE

 - New or Enhanced Feature

##### COMPONENT NAME

 - Other

### CODE REVIEW

**Comment by rooftopcellist** [comment]
I tested this make target locally by running:

```
$ IMG=quay.io/chadams/awx:test-multi-arch make docker-compose-buildx
```

image

File: Makefile [hunk 1]
```diff
@@ -585,6 +585,29 @@ docker-compose-build: Dockerfile.dev
...
+# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
+# To properly provided solutions that supports more than one platform you should use this option.
+PLATFORMS ?= linux/amd64,linux/arm64  # linux/ppc64le,linux/s390x
+.PHONY: docker-compose-buildx
+docker-compose-buildx: Dockerfile.dev ## Build and push docker image for the manager for cross-platform support
+	- docker buildx create --name project-v3-builder
+	docker buildx use project-v3-builder
+	- docker buildx build --push $(BUILD_ARGS) --platform=$(PLATFORMS) --tag ${IMG} -f Dockerfile.dev .
+	- docker buildx rm project-v3-builder
+
```
Comment (TheRealHaoLiu) [inline Makefile]: out of scope for this PR?

File: Makefile [hunk 2]
```diff
@@ -585,6 +585,21 @@ docker-compose-build: Dockerfile.dev
...
```
Comment (TheRealHaoLiu) [inline Makefile]: this doesnt belong in this PR either

Comment (rooftopcellist) [reply Makefile]: I broke these changes out to another PR:
* https://github.com/ansible/awx/pull/14774

File: .github/workflows/stage.yml [hunk 3]
```diff
@@ -86,27 +86,33 @@ jobs:
...
         working-directory: awx-operator
         run: |
-          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.version }} \
-                      --build-arg OPERATOR_VERSION=${{ github.event.inputs.operator_version }}" \
-          IMAGE_TAG_BASE=ghcr.io/${{ github.repository_owner }}/awx-operator \
-          VERSION=${{ github.event.inputs.operator_version }} make docker-build docker-push
+          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.default_awx_version }} \
+              --build-arg OPERATOR_VERSION=${{ github.event.inputs.version }}" \
+          IMG=ghcr.io/${{ github.repository_owner }}/awx-operator:${{ github.event.inputs.version }} \
+          make docker-buildx
```
Comment (TheRealHaoLiu) [inline .github/workflows/stage.yml]:
```suggestion
          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.version}} \
              --build-arg OPERATOR_VERSION=${{ github.event.inputs.version }}" \
          IMG=ghcr.io/${{ github.repository_owner }}/awx-operator:${{ github.event.inputs.version }} \
          make docker-buildx
```

Comment (TheRealHaoLiu) [reply .github/workflows/stage.yml]: i dont see `github.event.inputs.default_awx_version` being defined

File: .github/workflows/stage.yml [hunk 4]
```diff
@@ -86,27 +86,33 @@ jobs:
...
         working-directory: awx-operator
         run: |
-          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.version }} \
-                      --build-arg OPERATOR_VERSION=${{ github.event.inputs.operator_version }}" \
-          IMAGE_TAG_BASE=ghcr.io/${{ github.repository_owner }}/awx-operator \
-          VERSION=${{ github.event.inputs.operator_version }} make docker-build docker-push
+          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.version}} \
+              --build-arg OPERATOR_VERSION=${{ github.event.inputs.version }}" \
+          IMG=ghcr.io/${{ github.repository_owner }}/awx-operator:${{ github.event.inputs.version }} \
+          make docker-buildx
```
Comment (TheRealHaoLiu) [inline .github/workflows/stage.yml]:
```suggestion
          BUILD_ARGS="--build-arg DEFAULT_AWX_VERSION=${{ github.event.inputs.version}} \
              --build-arg OPERATOR_VERSION=${{ github.event.inputs.operator_version }}" \
          IMG=ghcr.io/${{ github.repository_owner }}/awx-operator:${{ github.event.inputs.operator_version }} \
          make docker-buildx
```

**Review by dmzoneill — APPROVED** [review]

**Pull request merged**